### PR TITLE
allow output path that is not the root

### PR DESCRIPTION
**Pull request by bdwain**
addresses #8 in a backwards compatible way. 

There was no way without a new option to see the relative path from the root output folder to your image output folder. So the easiest thing was to add a second option, but default its value to the value of outputFolder for the common case where you want everything at the root.

This should not affect anyone who is currently using rebase since it only makes a difference if you add in the outputBase option.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -104,16 +104,20 @@ export default function rebase(options = {}) {
 
           var inputFolder = path.dirname(path.resolve(input))
           var relativeToRoot = path.relative(path.dirname(fileSource), inputFolder).replace(/\\/g, "/")
+          var relativeOutputPath = path.relative(outputBase || outputFolder, outputFolder);
```
Comment (bdwain) [inline src/index.js]: if outputBase is not set, this will be empty and have no effect.

**Comment by bdwain** [comment]
any thoughts on this? I think it'd be nice just for the purposes of organizing our output folder and making it more human readable

**Comment by bdwain** [comment]
hi @swernerx any chance you could take a look at this one?

**Comment by swernerx** [comment]
Released as 1.0.0 :) Thank you for your contribution!

**Comment by bdwain** [comment]
thanks!

**Pull request merged**